core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -195,6 +198,21 @@ protected[controller] class RestAPIVersion_v1()(
     private val webexp = new WebActionsApi(Seq("experimental", "web"), WebApiDirectives.exp)
     private val web = new WebActionsApi(Seq("web"), WebApiDirectives.web)
 
+    /**
+     * Describes details of a particular API path.
+     */
+    private val info = JsObject(
+        "description" -> "OpenWhisk API".toJson,
+        "min_cli_version" -> whiskConfig(whiskVersionDate).toJson,
+        "min_recommended_cli_version" -> whiskConfig(whiskVersionDate).toJson,
```

Comment:
Hmm, I like the idea. Maybe the CLI should be keeping track of the backend versions that it works with instead of the controller doing that job. Should the controller know that a CLI even exists?